### PR TITLE
Added Buttons

### LINKED ISSUE #56: Keybase.io support
If possible, it would be nice, if there was a button for Keybase.io profile and also possibility to add keybase.txt file to be hosted on site as the website verification method for Keybase.io.

### LINKED ISSUE #90: Feature Request(s): Additional Links | STREAMLABS
Feature Request(s): Additional Links

Requesting

 - STREAMLABS_TIP

### LINKED ISSUE #92: Feature Request(s): Additional Links | TEESPRING
Feature Request(s): Additional Links

Requesting

 - TEESPRING

### LINKED ISSUE #98: Feature Request(s): Additional Links | XING
Feature Request(s): Additional Links

* Requesting

XING

### LINKED ISSUE #103: Feature Request(s): Additional Links | Session
Feature Request(s): Additional Links

Requesting

Session
(getsession.org)

### LINKED ISSUE #104: Feature Request(s): Additional Links |  Threema
Feature Request(s): Additional Links

Requesting

Threema
(https://threema.ch/)
Thanks for your work !

### LINKED ISSUE #106: Feature Request: Additional Links | OnlyFans
Feature Request: Additional Links

Requesting

OnlyFans
(https://onlyfans.com/)

Something for the female users ;)

### LINKED ISSUE #131: Feature Request: Add PrivateBin
Feature Request(s): Additional Links

Requesting

[PrivateBin](https://privatebin.net/)

### PR DESCRIPTION
**Pull request by KieranRobson**
# Proposed Changes

- Add Teespring - Closes #92 
- Add Xing - Closes #98 
- Add Keybase - Closes #56 
- Add Onlyfans - Closes #106 
- Add Session - Closes #103 
- Add Threema - Closes #104 
- Add Streamlabs - Closes #90 (No Good Logo SVG can be found)
- Add Privatebin - Closes #131 
- Edit CashApp to make pure white

image

Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [x] All my changes appear after other changes in each file
- [x] Included a screenshot (if adding a new button)
- [x] 🚀

### CODE REVIEW

**Comment by KieranRobson** [comment]
Forgot to change image after testing, Dont merge just yet haha

File: src/config.js
Comment (timothystewart6) [inline src/config.js:346]: I think these need to be:

```javascript
TEESPRING: nodeIsProduction ?  ? process.env.TEESPRING
          : process.env.RAZZLE_TEESPRING
```

basically use a different variable if not in production, otherwise this looks great!!!

**Review by timothystewart6 — CHANGES_REQUESTED** [review]
One small change to use the  `RAZZLE_` env, otherwise nice work!!!

**Comment by KieranRobson** [comment]
@timothystewart6 I got too into it I didn't realise I missed them haha 

Razzles added

**Review by timothystewart6 — APPROVED** [review]

**Comment by timothystewart6** [comment]
@KieranRobson   All good!  it's easy to overlook!  Thank you so much!  Nice work!!

**Comment by madvip3r** [comment]
I found weird bug:
Screenshot 2022-06-21 at 18-35-16 madviper

After adding `- KEYBASE=https://keybase.io/<username>` in `docker-compose.yml` and recreating container it's generate all other buttons, all linking to Keybase link.

**Comment by Choubakawa** [comment]
They use `process.env.KEYBASE` instead of their own env

**Comment by timothystewart6** [comment]
I think this was fixed by https://github.com/techno-tim/littlelink-server/pull/149

**Pull request merged**
